training_data_collector/main/chessdotcom/labelgame.py

```diff
@@ -0,0 +1,55 @@
+from selenium.webdriver.common.by import By
+import cv2
+
+from chessdotcom import actions
+from darknetconvert import convert_co_ords_to_darknet
+from flags import flags
+from image_enrichment import enrich_image
+
+def label_game(driver):
+    gameName = flags['game_id']
+    moveCount = actions.get_move_count(driver)
+    board_size = flags['min_board_size']
+
+    for x in range(1, moveCount + 1):
+        if board_size != flags['max_board_size']:
+            board_size += flags['board_size_increment']
+            # actions.change_board_size(driver, board_size)
+        else:
+            board_size = flags['min_board_size']
+        label_board(driver, x, gameName)
+        actions.next_move(driver)  
+    label_board(driver, x + 1, gameName)
+
+def label_board(driver, move_number, game_name):
+    move_name = 'move_' + str(move_number)
+    full_name = game_name + '_' + move_name
+    print(f"training_data/images/{flags['train_or_val']}/{full_name}.png")
+    actions.take_screenshot(driver, f"training_data/images/{flags['train_or_val']}/{full_name}.png")
+    label_element(driver, f"//*[@id='board-dailyGame-{flags['game_id']}']", full_name)
+    i = 1 
+    while(True):
```

Comment:
Bit of a code smell, you can use "os" or something to see if the files exist instead of just trying til it throws an error. I know this is like a side tool thing but definitely should note it as an issue in github at least if you don't wanna change it right now.

Reply:
I'll try tidy that up at some stage, haven't changed it yet.